docs/arch/adr-010-new-versioning-scheme.md

```diff
@@ -0,0 +1,193 @@
+
+### Choosing the Channel
+
+Our revised versioning scheme entails organising our offering into Channels that align with specific Kubernetes minor versions. As a result, users will be required to select the Channel to which they want their installer assigned.
+
+Although we won't initially provide a Channel for every Kubernetes minor version, we have arbitrarily selected Kubernetes minor version 1.24 as our starting point. At the time of writing, here are the Channels that we will be offering to our users:
+
+| Channel  | Kubernetes version installed by this Channel    |
+|----------|-------------------------------------------------|
+| v1.24    | Most up-to-date Kubernetes version for v1.24.x. |
+| v1.25    | Most up-to-date Kubernetes version for v1.25.x. |
+| v1.26    | Most up-to-date Kubernetes version for v1.26.x. |
+| v1.27    | Most up-to-date Kubernetes version for v1.27.x. |
+
+By drafting an installation file similar to the model given below, users can link their setup with a particular kURL Channel (_therefore they will be adhering to Kubernetes versions 1.27_)
+```yaml
+apiVersion: "cluster.kurl.sh/v1beta2"
+kind: "Installer"
+metadata:
+  name: "cluster"
+spec:
+  channel:  "1.27"
+```
+_We have decided to transition to the newer **v1beta2** variant. Existing users who prefer to continue working with the outdated format may still do so at their own discretion_
+
+#### Pinning the kURL version
+
+Another feature we aim to offer is the ability for users to fix (pin) their installation to a particular kURL installer version. The organisation of kURL installer releases follows the pattern `YYYY.MM.DD-x`. A complete list of kURL versions can be seen [here](https://github.com/replicatedhq/kURL/releases).
+
+For those seeking to pin their setup to a designated kURL installer version, the `installerVersion` field must be appropriately configured as shown in the example below.
+
+```yaml
+apiVersion: "cluster.kurl.sh/v1beta2"
+kind: "Installer"
+metadata:
+  name: "cluster"
+spec:
+  channel:  "1.27"
+  installerVersion: "v2023.04.24-0"
+```
+
+While it's certainly possible to omit the `installerVersion` configuration, doing so means relying upon the current most up-to-dated kURL installer version.
+
+### Add-on versions
+
+In order to maintain greater control over the installation process, we have decided to modify our approach to add-on selection. Going forward, we will no longer allow users to freely select any add-on on any version of their choosing. Instead, we will provide users with specific bundles of add-ons (from here on this ADR will call them _Bundles_), each of which will be available in different versions within a designated Channel.
+
+This is an example of a Bundle on its v1.0.0 version:
+
+| Add-on     | Version              |
+|------------|----------------------|
+| Containerd | 1.6.20               |
+| Flannel    | 0.21.5               |
+| OpenEBS    | 3.5.0.               |
+| Minio      | 2023-03-13T19-46-17Z |
```

Comment:
I know, there are more add-ons that this but I don't think that they are relevant to the example